<launch>
  <arg name="input_route_topic_name" default="/planning/mission_planning/route"/>
  <arg name="map_topic_name" default="/map/vector_map"/>

  <!-- vehicle info -->
  <arg name="vehicle_info_param_file" default="$(find-pkg-share vehicle_info_util)/config/vehicle_info.param.yaml"/>

  <node pkg="behavior_path_planner" exec="behavior_path_planner" name="behavior_path_planner" output="screen">
    <remap from="~/input/route" to="$(var input_route_topic_name)"/>
    <remap from="~/input/vector_map" to="$(var map_topic_name)"/>
    <remap from="~/input/perception" to="/perception/object_recognition/objects"/>
    <remap from="~/input/odometry" to="/localization/kinematic_state"/>
    <remap from="~/input/accel" to="/localization/acceleration"/>
    <remap from="~/output/path" to="path_with_lane_id"/>
    <remap from="~/input/scenario" to="/planning/scenario_planning/scenario"/>

    <param from="$(find-pkg-share behavior_path_planner)/config/side_shift/side_shift.param.yaml"/>
    <param name="bt_tree_config_path" value="$(find-pkg-share behavior_path_planner)/config/behavior_path_planner_tree.xml"/>
    <param from="$(var vehicle_info_param_file)"/>
  </node>
</launch>
